ModuleFourTasks/Task3/UserTaskService.cs

```diff
@@ -12,25 +12,29 @@ public UserTaskService(IUserDao userDao)
             _userDao = userDao;
         }
 
-        public int AddTaskForUser(int userId, UserTask task)
+        public void AddTaskForUser(int userId, UserTask task)
         {
             if (userId < 0)
-                return -1;
+            {
+                throw new ArgumentException("Invalid userId");
+            }
 
             var user = _userDao.GetUser(userId);
             if (user == null)
-                return -2;
+            {
+                throw new ArgumentException("User not found");
```

Comment:
Please, create a custom exception `UserNotFoundException`. [How to create user-defined exceptions](https://docs.microsoft.com/en-us/dotnet/standard/exceptions/how-to-create-user-defined-exceptions)

Reply:
Fixed in cbd4ea3.